src/calendar-heatmap.js

```diff
@@ -3,23 +3,52 @@ function calendarHeatmap() {
   // defaults
   var width = 750;
   var height = 110;
-  var legendWidth = 150;
+  var legendWidth = 165;
   var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
-  var days = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
+  // var days = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
+  var days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
   var selector = 'body';
-  var SQUARE_LENGTH = 11;
-  var SQUARE_PADDING = 2;
+  var SQUARE_LENGTH = 12;
+  var SQUARE_PADDING = 1;
   var MONTH_LABEL_PADDING = 6;
   var now = moment().endOf('day').toDate();
   var yearAgo = moment().startOf('day').subtract(1, 'year').toDate();
   var data = [];
-  var colorRange = ['#D8E6E7', '#218380'];
+  var colorRange = ['#bcd9d8', '#218380'];
   var tooltipEnabled = true;
   var tooltipUnit = 'contribution';
   var legendEnabled = true;
   var onClick = null;
+  var onMouseOver = null;
+  var onMouseOut = null;
+  var dateRange;
+
+  var blankColor = '#fbfcfc';
+  var zeroColor = '#fbfcfc';
+  var futureColor = '#f1f2f3';
 
   // setters and getters
+  chart.dateRange = function(value) {
+    if (!arguments.length) {
+      return dateRange; }
+    dateRange = value;
+    return chart;
+  };
+
+  chart.width = function(value) {
+    if (!arguments.length) {
+      return width; }
+    width = value;
+    return chart;
+  };
+
+  chart.height = function(value) {
+    if (!arguments.length) {
+      return height; }
```

Comment:
In the above three functions, change the formatting of this condition to match the rest of the file

```javascript
if (!arguments.length) { return height; }
```